src/common/reduction/test/ClusterTest.cpp

```diff
@@ -133,6 +133,12 @@ TEST_F(ClusterTest, CoordsMass) {
   EXPECT_EQ(cluster.coord_center(), 2);
 }
 
+TEST_F(ClusterTest, CoordMass2) {
+  cluster.insert(Hit{0, 2, 3, 0});
+  EXPECT_TRUE(cluster.valid());
+  EXPECT_EQ(cluster.coord_mass2(), 3*3 * 2);
+}
+
```

Comment:
Looks good for thefirst test. Here are some suggestions for improvement.
I suggest parametrising this test. Perhaps as a nested for loop running through
many (cord, mass) values. Also try to find the limits - are the inputs uint8_t's or ?
What happens with 'invalid' input (0,0,0,0)?